js/functional/palindrome.js

```diff
@@ -28,7 +52,21 @@ function isPalindromeUsingLoop(value) {
  */
 function isPalindromeUsingRecursive(value, index = 0) {
   // write your code here
-  return true;
+  // Remove non-alphabetic characters from the value and convert it to lowercase
+  value = value.replace(/[^a-zA-Z]/g, "").toLowerCase();
```

Comment:
I guess you want to sanitize the incoming value by using string.replace() API, but the problem is your way to do that is so repetitive. Also, I see the difference sanitize logic between the loop, reverse, and recursive. Please revise this part.

Reply:
The logic on cleanString() function is good, just make sure to use a better description at cleanString() function.